src/main/java/bookopedia/logic/commands/ViewCommand.java

```diff
@@ -37,12 +38,19 @@ public CommandResult execute(Model model) throws CommandException {
             throw new CommandException(Messages.MESSAGE_INVALID_PERSON_DISPLAYED_INDEX);
         }
         Person deliveryToView = lastShownList.get(index.getZeroBased());
+        StringBuilder parcelsToPrint = new StringBuilder("");
+        for (Parcel p : deliveryToView.getParcels()) {
+            parcelsToPrint.append(p.toString());
+            parcelsToPrint.append("\n");
```

Comment:
```suggestion
            parcelsToPrint.append(" ");
```
Might want to append with a SPACE instead, in case there are alot of parcels

Reply:
Ah ok will do that